proto/desmos/posts/v1/query.proto

```diff
@@ -0,0 +1,123 @@
+syntax = "proto3";
+package desmos.posts.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+import "desmos/posts/v1/models.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v3/x/posts/types";
+
+// Query defines the gRPC querier service
+service Query {
+  // Posts queries all the posts inside a given subspace
+  rpc Posts(QueryPostsRequest) returns (QueryPostsResponse) {
+    option (google.api.http).get = "/desmos/posts/v1/{subspace_id}/posts";
+  }
+
+  // Post queries for a single post inside a given subspace
+  rpc Post(QueryPostRequest) returns (QueryPostResponse) {
+    option (google.api.http).get =
+        "/desmos/posts/v1/{subspace_id}/posts/{post_id}";
+  }
+
+  // PostAttachments queries the attachments of the post having the given id
+  rpc PostAttachments(QueryPostAttachmentsRequest)
+      returns (QueryPostAttachmentsResponse) {
+    option (google.api.http).get =
+        "/desmos/posts/v1/{subspace_id}/posts/{post_id}/attachments";
+  }
+
+  // PollAnswers queries the answers for the poll having the given id
+  rpc PollAnswers(QueryPollAnswersRequest) returns (QueryPollAnswersResponse) {
+    option (google.api.http).get = "/desmos/posts/v1/{subspace_id}/posts/"
+                                   "{post_id}/polls/{poll_id}/answers";
+  }
+
```

Comment:
`PollTallyResults` is missing.

Reply:
This has been removed because poll tally results are now directly placed inside the `Poll` object, just like the Cosmos SDK handles a proposal's tally results

Reply:
Can you update the adr for it?

Reply:
Done :heavy_check_mark: